homeassistant/components/dynalite/__init__.py

```diff
@@ -70,24 +63,23 @@
 async def async_setup_entry(hass, entry):
     """Set up a bridge from a config entry."""
     LOGGER.debug("__init async_setup_entry %s", entry.data)
```

Comment:
Please start logging messages with capital letter. We don't need to mention what module we're in, as that will be logged by default.

Reply:
is this also relevant for debug messages or just INFO and above? Will start cleaning the logs

Also, I don't mention the module. This would be "homeassistant.components.dynalite", right? Just want to debug log the entry point to a specific function if i have to trace stuff

Reply:
all logs now capitalized